authors/apps/articles/renderers.py

```diff
@@ -46,3 +46,15 @@ def render(self, data, media_type=None, renderer_context=None):
             return json.dumps({'Comment': data})
 
         return json.dumps({'Comments': data})
+
+
+class ReportJSONRenderer(JSONRenderer):
+    charset = 'utf-8'
+
+    def render(self, data, media_type=None, renderer_context=None):
+
+        # Finally, we can render our data under the "profile" namespace.
+        return json.dumps({
+            'report details': data
```

Comment:
@BrianSerem, you can use a better key instead of `reporter details`, also, it is best to use keys that are snake case as JS makes it easy to work with them as objects.

Reply:
Okay let me work on these changes